packages/app-store/tandemvideo/api/callback.ts

```diff
@@ -60,7 +60,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     responseBody.expiry_date = Math.round(Date.now() + responseBody.expires_in * 1000);
     delete responseBody.expires_in;
 
-    createOAuthAppCredential({ appId: "tandem", type: "tandem_video" }, responseBody, req);
+    await createOAuthAppCredential({ appId: "tandem", type: "tandem_video" }, responseBody, req);
```

Comment:
@joeauyeung It returns a promise and because it wasn't awaited, any error thrown by the method was uncaught. It fixes that.